_includes/js/users.md

```diff
@@ -450,7 +450,7 @@ To create a link to an un-authenticated user (for example in cloud code), option
 
 ```javascript
 const myAuthData = {
-  id: xzx5tt123,
+  id: xzx5tt123,  // The id key is required in the authData-object. Otherwise Parse server will throw the Error 252 'This authentication method is unsupported.'
```

Comment:
```suggestion
  id: xzx5tt123,  // The id key is required in the authData-object. Otherwise Parse Server will throw the Error 252 'This authentication method is unsupported'.
```